src/scicookie/profiles/base.yaml

```diff
@@ -289,3 +289,15 @@ git_main_branch:
   type: "text"
   default: main
   visible: true
+
+continuos_integration:
+  message: Select one option for Continuous Integration
+  help: "For more information, check:\n  https://osl-incubator.github.io/scicookie/guide/#continuous-integration"
+  type: single-choice
```

Comment:
As we discussed some days ago, I think that this should be multiple choice instead of single choice  .. although it is not common to use multiples CI .. but it cana happen